relay-server/src/services/store.rs

```diff
@@ -302,10 +302,26 @@ impl StoreService {
                         .items()
                         .map(|item| item.ty().as_str())
                         .collect::<Vec<_>>();
+                    let attachment_types = envelope
+                        .items()
+                        .map(|item| {
+                            item.attachment_type()
+                                .map(|t| t.to_string())
+                                .unwrap_or_default()
+                        })
+                        .collect::<Vec<_>>();
 
                     relay_log::with_scope(
                         |scope| {
                             scope.set_extra("item_types", item_types.into());
+                            scope.set_extra("attachment_types", attachment_types.into());
+                            if other == &ItemType::FormData {
+                                let payload = item.payload();
+                                let form_data_keys = FormDataIter::new(&payload)
```

Comment:
This parses the payload inside the store service, which is not ideal but it only concerns a very small portion of traffic. We can remove it once the issue is resolved.